components/event/EventCard.js

```diff
@@ -43,7 +43,7 @@ export default function EventCard({ event, username }) {
                   key={event.url}
                   target="_blank"
                   rel="noreferrer"
-                  className="text-lg lg:text-xl tracking-wide font-medium capitalize"
+                  className="text-lg lg:text-xl tracking-wide font-medium capitalize hover:underline"
```

Comment:
This already exists in the Link component, maybe the component needs to be updated so it always appends `hover:underline` rather than replaces it with the incoming classes? https://github.com/EddieHubCommunity/LinkFree/blob/main/components/Link.js#L10